scijava/scijava-ops/src/main/java/org/scijava/ops/function/Computers.java

```diff
@@ -431,7 +431,7 @@ default void accept(final I1 in1, final I2 in2, final I3 in3, final I4 in4, fina
 		Consumers.Arity17<I1, I2, I3, I4, I5, I6, I7, I8, I9, I10, I11, I12, I13, I14, I15, I16, O>
 	{
 
-		void compute(I1 in1, I2 in2, I3 in3, I4 in4, I5 in5, I6 in6, I7 in7, I8 in8, I9 in9, I10 in10, I11 in11, I12 in12, I13 in13, I14 in14, I15 in15, I16 in16, @Mutable O out);
+		void compute(I1 in1, I2 in2, I3 in3, I4 in4, I5 in5, I6 in6, I7 in7, I8 in8, I9 in9, I10 in10, I11 in11, I12 in12, I13 in13, I14 in14, I15 in15, I16 in16, @Container O out);
```

Comment:
Overall I think `@Mutable` and `@Container` are more descriptive, but I don't understand why some compute methods have an `@Container` but others don't?

e.g. https://github.com/scijava/incubator/pull/27/files#diff-199835d92f5a48202497fb4a82748776e19fe8d432e17c609dc462708ce36c11R558

Reply:
@hinerm if I remember correctly, we only need these annotations on our functional interface methods (since this is where our API will pull them from). Your link goes to a method that is overriding the interface's method, thus it is probably best to leave the annotation off.

Do you have a rule for annotation application that you would prefer?

Reply:
@gselzer No that makes sense. I just noticed that the implementation methods had previously been annotated `@Mutable` so the drop in annotation stood out.

Reply:
Gotcha, thanks for pointing it out, but also that it is a good change (I think you agree with me on this, more just want to record my thoughts). I would personally want these annotations required (even allowable) in as few places as possible because it is:
* annoying to have to remember to put them there
* more work to change if/when we decide to alter the annotations/add new functional interfaces and support for them